types/sharedb/tslint.json

```diff
@@ -0,0 +1,79 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "adjacent-overload-signatures": false,
```

Comment:
Is there a particular reason all of these rules are disabled?

Reply:
This should be fixed. I had to disable dt-header (because ShareDB's version tag includes "beta" - v1.0.0-beta.11) and no-declare-current-package (because ShareDB includes two modules - one for clients and one for servers)